Formula/openmsx.rb

```diff
@@ -20,31 +19,42 @@ class Openmsx < Formula
     sha256 cellar: :any, high_sierra: "86fff1a90fff96cb0398184ef7ebdeb804edda4c9d34ee5e7278159df64b10e3"
   end
 
+  depends_on "python@3.9" => :build
```

Comment:
I believe this is only used during build: https://openmsx.org/manual/compile.html#tools